x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -1074,6 +1075,48 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
       });
     });
 
+    describe('Tabs - alerts linked to case', () => {
+      before(async () => {
+        await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/rule_registry/alerts');
+        await cases.navigation.navigateToApp();
+        const theCase = await cases.api.createCase();
+        await cases.casesTable.waitForCasesToBeListed();
+        await retry.try(async () => {
+          await cases.api.createAttachment({
+            caseId: theCase.id,
+            params: {
+              type: AttachmentType.alert,
+              alertId: ['NoxgpHkBqbdrfX07MqXV'],
+              index: '.alerts-observability.apm.alerts',
+              rule: { id: 'id', name: 'name' },
+              owner: theCase.owner,
+            },
+          });
+        });
+      });
+
+      after(async () => {
+        await cases.api.deleteAllCases();
+        await esArchiver.unload('x-pack/test/functional/es_archives/rule_registry/alerts/');
+      });
+
+      beforeEach(async () => {
+        await cases.navigation.navigateToApp();
+        await cases.casesTable.goToFirstListedCase();
+        await header.waitUntilLoadingHasFinished();
+      });
+
+      it('should show the right amount of alerts linked to a case', async () => {
+        const visibleText = await testSubjects.getVisibleText('case-view-alerts-stats-badge');
+        expect(visibleText).to.be('1');
+      });
+
+      it('should render the alerts table when opening the alerts tab', async () => {
+        await testSubjects.click('case-view-tab-title-alerts');
+        await testSubjects.existOrFail('alertsStateTableEmptyState');
```

Comment:
I think `alertsStateTableEmptyState ` is visible when there are no alerts, right? Shouldn't  this be `miisngOrFail`?

Reply:
When there is no alert we don't render the alerts table, we render this component
![image](https://github.com/user-attachments/assets/1026a40f-1c16-4352-9911-a1826dc699bc)
Here is [code](https://github.com/elastic/kibana/blob/c7cef4c95ba01b21ba8c73900242b9c117b8142d/x-pack/platform/plugins/shared/cases/public/components/case_view/components/case_view_alerts.tsx#L77)

In dev/prod we would see the alert in the alerts table but here we just see an empty one because of the data we are using. We would need to update the data, which I think means adding the rule data specifics, but not sure. I stopped here as for this test it's enough to ensure that the alerts table is there, we don't really care about its content. That is, hopefully, being tested in another test

Reply:
Got it, in that case may be we could remove this test and keep just the first one?

Reply:
I prefer to make sure that the table is being rendered just in case and because it was specifically mentioned in the issue